be/src/olap/row_cursor.h

```diff
@@ -73,18 +73,27 @@ class RowCursor {
     OLAPStatus init(const TabletSchema& schema,
                     const std::vector<uint32_t>& columns);
 
+    OLAPStatus init(const std::vector<TabletColumn>& schema,
+                    const std::vector<uint32_t>& columns);
+
     // 用传入的key的size来初始化
     // 目前仅用在拆分key区间的时候
     OLAPStatus init_scan_key(const TabletSchema& schema,
                              const std::vector<std::string>& keys);
 
+    OLAPStatus init_scan_key(const std::vector<TabletColumn>& schema,
```

Comment:
If you call this function, you will need all columns of schema. There is no necessary to overload with std::vector<TabletColumn>& schema parameter.

Reply:
OK